requirements-test-libraries.txt

```diff
@@ -100,6 +100,7 @@ soundfile==0.10.3.post1; sys_platform != 'linux'
 yt-dlp==2022.5.18
 limits==2.6.3
 great-expectations==0.15.9
+tensorflow==2.9.1

```

Comment:
That's an ironic place to put tensorflow - anywhere near the words *great expectations*.

Reply:
We need to find a package named *blood-sweat-and-carnage* to stick it next to instead.

Reply:
> We need to find a package named _blood-sweat-and-carnage_ to stick it next to instead.

That sounds like an alias for PyInstaller.